build-system/tasks/server-tests.js

```diff
@@ -68,17 +68,17 @@ async function getExpectedOutput(inputFile) {
 
 /**
  * Extracts the JS transform for a test from its transform file.
- *
  * @param {string} inputFile
+ * @param {JSON} extraOptions
```

Comment:
Is there really a `JSON` type?

Reply:
This one, I'm not 100% sure about.
@kevinkimball would you happen to know?